### ISSUE TITLE
CI error

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-web#1189]
See https://github.com/fractal-analytics-platform/fractal-web/actions/runs/27937847249/job/82663766575

<img width="2593" height="2004" alt="Image" src="https://github.com/user-attachments/assets/484f06fb-2a6a-4a44-8c15-4cd21d1ed2fd" />

**Comment by zonia3000** [comment]
The error message was:

> Element has insufficient color contrast of 1.22 (foreground color: #e2e3e4, background color: #f9fafb, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
> `"html": "<div data-id=\"kxb560xo\" id=\"ss-cp0n2l30-kxb560xo\" class=\"ss-option\" role=\"option\" aria-selected=\"false\">admin@fractal.xy</div>"`

The failure happened sporadically. This was a false positive caused by the fact that the accessibility check was performed while the dropdown menu was closing. The contrast is fine, but it was computed while the element was fading out.

I added the following line at the end of the test to wait the element disappearing, hoping to mitigate the issue:

```js
await expect(page.getByRole('option', { name: 'admin@fractal.xy' })).not.toBeVisible();
```
